Userland/Applets/ClipboardHistory/ClipboardHistoryModel.cpp

```diff
@@ -10,6 +10,7 @@
 #include <AK/NumberFormat.h>
 #include <AK/StringBuilder.h>
 #include <LibConfig/Client.h>
+#include <LibCore/DateTime.h>
```

Comment:
We don't need to include this in both ClipboardHistoryModel.h and ClipboardHistoryModel.cpp, just in the .h is fine

Reply:
Oops, right.